helpers/kubernetes/tailer/setup.go

```diff
@@ -86,22 +88,45 @@ func FetchLogs(ctx context.Context, logChan chan ContainerLogLine, wg *sync.Wait
 		return true
 	}
 
+	logger.Info("filter pods, containers")
+
 	for _, pod := range podList.Items {
-		for _, c := range pod.Spec.Containers {
+		for _, c := range pod.Spec.InitContainers {
```

Comment:
I wonder if it's guaranteed that `InitContainers` come always in the right order. By trying out it seems so and it's also logical to be the case, since they are supposed to be executed in order. I guess Kubernetes is using this very same slice to execute them in order.

Reply:
Yep.